src/components/CodeShower.vue

```diff
@@ -130,11 +130,11 @@ for(let screenI=0; screenI<this.screensObjects.length; screenI++) {
 }
 
 text+=`
+  ActiveScreen = 0
```

Comment:
Approved because this will remove a warning (even though the function call was _already_ after the variable declaration).